drivers/wifi/nrf700x/osal/fw_if/umac_if/inc/fw/host_rpu_umac_if.h

```diff
@@ -2889,7 +2892,7 @@ struct umac_display_results {
 	unsigned short capability;
 	struct nrf_wifi_signal signal;
 	unsigned char twt_support;
-	unsigned char reserved2;
+	unsigned char pmf_flag;
```

Comment:
Can we stick to one terminology...if we are calling it mfp everywhere then it is better to call this too as mfp_flag.

Reply:
Done.